assets/xml/objects/object_skb.xml

```diff
@@ -1,64 +1,65 @@
-        <Limb Name="object_skb_Standardlimb_005DE0" Type="Standard" EnumName="OBJECT_SKB_LIMB_03" Offset="0x5DE0" />
-        <Limb Name="object_skb_Standardlimb_005DEC" Type="Standard" EnumName="OBJECT_SKB_LIMB_04" Offset="0x5DEC" />
-        <Limb Name="object_skb_Standardlimb_005DF8" Type="Standard" EnumName="OBJECT_SKB_LIMB_05" Offset="0x5DF8" />
-        <Limb Name="object_skb_Standardlimb_005E04" Type="Standard" EnumName="OBJECT_SKB_LIMB_06" Offset="0x5E04" />
-        <Limb Name="object_skb_Standardlimb_005E10" Type="Standard" EnumName="OBJECT_SKB_LIMB_07" Offset="0x5E10" />
-        <Limb Name="object_skb_Standardlimb_005E1C" Type="Standard" EnumName="OBJECT_SKB_LIMB_08" Offset="0x5E1C" />
-        <Limb Name="object_skb_Standardlimb_005E28" Type="Standard" EnumName="OBJECT_SKB_LIMB_09" Offset="0x5E28" />
-        <Limb Name="object_skb_Standardlimb_005E34" Type="Standard" EnumName="OBJECT_SKB_LIMB_0A" Offset="0x5E34" />
-        <Limb Name="object_skb_Standardlimb_005E40" Type="Standard" EnumName="OBJECT_SKB_LIMB_0B" Offset="0x5E40" />
-        <Limb Name="object_skb_Standardlimb_005E4C" Type="Standard" EnumName="OBJECT_SKB_LIMB_0C" Offset="0x5E4C" />
-        <Limb Name="object_skb_Standardlimb_005E58" Type="Standard" EnumName="OBJECT_SKB_LIMB_0D" Offset="0x5E58" />
-        <Limb Name="object_skb_Standardlimb_005E64" Type="Standard" EnumName="OBJECT_SKB_LIMB_0E" Offset="0x5E64" />
-        <Limb Name="object_skb_Standardlimb_005E70" Type="Standard" EnumName="OBJECT_SKB_LIMB_0F" Offset="0x5E70" />
-        <Limb Name="object_skb_Standardlimb_005E7C" Type="Standard" EnumName="OBJECT_SKB_LIMB_10" Offset="0x5E7C" />
-        <Limb Name="object_skb_Standardlimb_005E88" Type="Standard" EnumName="OBJECT_SKB_LIMB_11" Offset="0x5E88" />
-        <Limb Name="object_skb_Standardlimb_005E94" Type="Standard" EnumName="OBJECT_SKB_LIMB_12" Offset="0x5E94" />
-        <Limb Name="object_skb_Standardlimb_005EA0" Type="Standard" EnumName="OBJECT_SKB_LIMB_13" Offset="0x5EA0" />
-        <Skeleton Name="object_skb_Skel_005EF8" Type="Normal" LimbType="Standard" LimbNone="OBJECT_SKB_LIMB_NONE" LimbMax="OBJECT_SKB_LIMB_MAX" EnumName="ObjectSkbLimb" Offset="0x5EF8" />
-        <Animation Name="object_skb_Anim_0064E0" Offset="0x64E0" />
+        <Animation Name="gStalchildStandUpAnim" Offset="0x3584" />
+        <Texture Name="gStalchildPelvisTex" OutName="stalchild_pelvis" Format="rgba16" Width="16" Height="16" Offset="0x35A0" />
+        <Texture Name="gStalchildRibcageTex" OutName="stalchild_ribcage" Format="rgba16" Width="8" Height="16" Offset="0x37A0" />
+        <Texture Name="gStalchildSkullTex" OutName="stalchild_skull" Format="rgba16" Width="8" Height="8" Offset="0x38A0" />
+        <Texture Name="gStalchildNoseTex" OutName="stalchild_nose" Format="rgba16" Width="8" Height="8" Offset="0x3920" />
+        <Texture Name="gStalchildSingleBoneTex" OutName="stalchild_single_bone" Format="rgba16" Width="8" Height="8" Offset="0x39A0" />
```

Comment:
Wasn't sure what name to use here. This tex is used in the arms, legs, and spine.

Reply:
OoT has it named as "gStalchildBoneTex"

Reply:
Changed to the OoT version in latest commit.